### ISSUE TITLE
Screen Reader+AXE-Core Adjustments for 508 Spot Check

### ISSUE THREAD

**Issue by jason-gcio** [department-of-veterans-affairs/va.gov-team#4013]
## Story
As a speech input user I need to be able to:
1) navigate the page by speaking the visible text labels of menus, links, and buttons that appear on the screen so I may navigate the content reliably, and;
2) navigate the heading hierarchy so I may understand the content of the page, and;
3) understand the content of the page without the insertion of random character strings (when an icon/image fails to load) so that I may know what is relevant for me.

Also, as a member of the eBenefits team, I want to ensure that the features I produce are at the highest standard and leverage the most assistance to as many Veterans as possible when using a VA.gov in a browser.

## Considerations, Tasks, and Possible Solutions
- **Per Jennifer Strickland with the 508 team**, we need to ensure our links have their visible text as part of their accessible name.
  - For more context and a possible solution, please see [#3632](https://github.com/department-of-veterans-affairs/va.gov-team/issues/3632).
- **Per Jennifer Strickland with the 508 team**, we need to ensure the order of headings are semantically correct and simply increase by one.
  - For more context, please see [#3633](https://github.com/department-of-veterans-affairs/va.gov-team/issues/3633).
  - Our `h1` is followed by an `h3` and needs to be changed to an `h2`
  - The new `h2` can have its look and feel be unaffected by using a utility class:
```HTML
<h2 class="vads-u-font-family--sans vads-u-margin-y--1">Individual disability ratings</h2>
```
- **Per Jennifer Strickland with the 508 team**, decorative icons are generally suppressed so the inserted text doesn't get picked up by the screen reader.
    - For more context, please see [#3683](https://github.com/department-of-veterans-affairs/va.gov-team/issues/3683).
  - Add `aria-hidden="true" role="image"` attributes to the `<i>` element, like this:
```HTML
 <i class="fas fa-medal vads-u-margin-right--0p5" aria-hidden="true" role="img"></i>
```
## Acceptance Criteria
- [x] Links have been updated to have their visible text as part of their name
- [x] Headings start with `h1` and sequentially decrease by one as the page progresses
- [x] Decorative icons are suppressed for the screen reader
- [x] Unit tests have been updated

**Comment by micahchiang** [comment]
PR here - https://github.com/department-of-veterans-affairs/vets-website/pull/11267

**Comment by jason-gcio** [comment]
By decrease, I meant increase in number but get smaller in size.  Sorry for that confusion!
